fplll/pruner/pruner.h

```diff
@@ -124,10 +188,18 @@ template <class FT>
 void prune(/*(input)output*/ PruningParams &pruning,
            /*inputs*/
            const double enumeration_radius, const double preproc_cost, const vector<double> &gso_r,
-           const double target       = .9,
+           const double input_target = .9,
```

Comment:
Why `input_target`?

Reply:
Changed back to target. The purpose was: the input_target could be changed to something else in this->target after some sanity check.